include/slang.h

```diff
@@ -872,6 +872,7 @@ extern "C"
             DisableSourceMap,       // bool
             UnscopedEnum,           // bool
             PreserveParameters,       // bool: preserve all resource parameters in the output code.
+            EmbedDXIL,              // bool
```

Comment:
This will break binary compatibility, new enums should be added at the bottom of the type.